python/pyspark/pandas/tests/indexes/test_base.py

```diff
@@ -1542,6 +1540,19 @@ def test_union(self):
                     pidx2.union(pd.Series([1, 2, 3, 4, 3, 4, 3, 4])),
                     almost=True,
```

Comment:
Wouldn't the test now fail with pandas lower than 1.3? We should at least skip or make the tests pass manually in other versions